compiler/one-cmds/one-init

```diff
@@ -235,16 +284,40 @@ def _generate():
         pass  # NYI
 
     def _gen_import():
-        pass  # NYI
+        section = f'one-import-{model_type}'
```

Comment:
Q) where is `model_type` from?

Reply:
Ah, `_gen_import` is inner function of `_generate`... I'm not good with python and learned now about this :)